README.md

```diff
@@ -36,8 +36,8 @@ This is a non-profit project. We don't plan on monetizing the app, not even inse
 
 The collaborative features are being engineered with privacy in mind; users will not be uniquely identified. For example, when submitting information during trips, a random ID will be generated at the start of the trip, used for its duration, and destroyed once the user leaves the network.
 
-Maintaining the servers for the online components incurs costs that are paid out of our own pockets and eventually any proceedings from gbl08ma's for-profit (but not profitable) projects such as [tny.im](https://tny.im).
+Maintaining the servers for the online components incurs costs that are paid with the help of our patrons but also out of our own pockets and eventually any proceedings from gbl08ma's for-profit (but not profitable) projects such as [tny.im](https://tny.im).
```

Comment:
Remove this entire paragraph and move the sentence "Maintaining the servers for the online components incurs costs that are paid with the help of our patrons" to after "by becoming our patron on Patreon".